flow/TLSConfig.actor.cpp

```diff
@@ -64,6 +65,15 @@ std::vector<std::string> LoadedTLSConfig::getVerifyPeers() const {
 	return { "Check.Valid=1" };
 }
 
+bool LoadedTLSConfig::getDisablePlainTextConnection() const {
+	std::string envDisablePlainTextConnection;
+	if (platform::getEnvironmentVar("FDB_TLS_DISABLE_PLAINTEXT_CONNECTION", envDisablePlainTextConnection)) {
+		return boost::lexical_cast<bool>(envDisablePlainTextConnection);
```

Comment:
This could throw a [bad_lexical_cast](https://www.boost.org/doc/libs/1_82_0/doc/html/boost_lexical_cast/synopsis.html#boost_lexical_cast.synopsis.bad_lexical_cast) and we don't catch those in FDB/fdbcli, so it would crash the process.
It could be more graceful to catch and log/print the error.
Alternatively you could use `try_lexical_convert`.